apps/newsletters-api/src/register-ui-server.ts

```diff
@@ -1,32 +1,60 @@
 import fs from 'fs';
 import path from 'path';
-import { fastifyStatic } from '@fastify/static';
-import type { FastifyInstance, RouteHandlerMethod } from 'fastify';
+import type {
+	Express,
+	Request,
+	Response,
+} from 'express'
+import { static as serveStatic } from 'express';
 
-export function registerUIServer(app: FastifyInstance) {
+
+const routeMap = {
+	'/': [
+		'',
+		'/templates',
```

Comment:
Just an enquiry - why is `/templates` added to the `routeMap` differently from other endpoints like `/launched` and `/drafts`?

In the Fastify code, it appears to have the same pattern as others?

Reply:
I guess I was following the structure used in the [UI application](https://github.com/guardian/newsletters-nx/blob/main/apps/newsletters-ui/src/main.tsx) to define react-router routes - the `/templates` page is a subpath of the "homeRoute" `RouteObject` rather than being on its own `RouteObject`. I think it was done this way because `/templates` is a single page with no further routing off it.

From the server point of view, it would have made more sense to have done `/templates` like `/launched` and `/drafts`, but its equivalent since the routeMap is effectively flattened into the list of paths to serve the Ui to.

It would be good to clean this up on a future PR - if we add express routers, it would be nice if the client-side routes corresponded to them.